profiles/common/modules/custom/tmgmt_poetry/README.md

```diff
@@ -31,80 +31,44 @@ Table of content:
-
-- Once green light has been received from DG COMM, the feature needs to be
-activated by your FPFIS maintenance team.  Create a support ticket in
-[Jira's MULTISITE project]
-(https://webgate.ec.europa.eu/CITnet/jira/secure/RapidBoard.jspa) (use the blue
-button labelled *Create*) explaining the details and deadlines for your request:
-  - The ticket should contain the title 'Poetry activation request',
-  - The ticket should include the name of your project,
-  - Once maintenance team confirms the feature is ready to be used on
-    Playground and/or Production you can configure your connector.
-  - It is proved helpful to have the deadline crafted in the jira ticket title.
+:hand: You cannot enable DGT-Connector feature using feature sets.
 
-[Go to top](#table-of-content)
-
-### Configure the DGT connector:
-Once the module is enabled and the settings properly set, the webmaster can
-proceed with the module's configuration.
-
-Edit the translator labelled "DGT Connector (auto created)".
-
-In order to do this, navigate to:
-``` Configuration->Regional and Language->Translation management Translator ```.
-
-or go to:
-``` admin/config/regional/tmgmt_translator/manage/poetry_en ```.
-
- - Translator settings: [x] Auto accept finished translations
-   - Check this if you don't want to review a translation before publishing it.
-
- - Translator plugin:
-   - This is shown for information, please do not change it.
-
- - DGT Connector plugin settings:
-   - You should see 'Main "poetry_service" variable is properly set.' if you
-   have correctly followed the steps above. Otherwise, get back and check what
-   you forgot!
-   - Counter: you do not need to fill this. The counter is auto generated.
-   - Requester code: must always be *WEB* [See DGT reference explanation]
-   (#dgt-reference-number).
-   - Organization responsible, Author & requester: check the example provided.
-   - Contact usernames: should be the *username* (you connect to the network,
-  ecas or the proxy with your username) of the persons in charge of the request.
-  This is important as only these persons can view translation details in the
-  web app.
-   - DGT contacts: The email address of your contact person at DGT. They will
-   receive feedback on the translations you want to comment. That information
-   should have been provided to you by DGCOMM.
+- If approved by CEM, a JIRA issue will have been created in the [project 
+MULTISITE](https://webgate.ec.europa.eu/CITnet/jira/secure/RapidBoard.jspa) 
+- CEM will enable the feature for you.
+- The maintenance team will setup the DGT-Connector 
+(adaptation of "settings.php").
 
 [Go to top](#table-of-content)
 
 ## Maintenance staff (FPFIS staff)
 :construction: Only maintenance team can enable the DGT translator.
-### Enable the feature
-
-The module is included in the platform-dev sources. Run the drush command
-```drush en tmgmt_poetry```.
+### The feature
 
+Make sure the 
```

Comment:
I think the sentence is incomplete: "Make sure the Update the settings.php of the project.", no? That should not be "Make sure that the project's _"settings.php"_ is correctly updated"?